app/controllers/houston/dashboards/staging_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Houston::Dashboards::StagingController < ApplicationController
   layout "houston/dashboards/dashboard"
-  STAGING_PROJECTS = %w(members unite ledger bsb)
+  STAGING_PROJECTS = %w(members unite ledger bsb members-dav)
```

Comment:
Instead of defining `STAGING_PROJECTS` and looping through them to call `github.list_issues`, just use [org_issues](http://www.rubydoc.info/github/pengwynn/octokit/Octokit%2FClient%2FIssues%3Aorg_issues). We can probably safely assume that _any_ pull request in our organization with **test-needed** or **on-staging** belongs on this dashboard.